index.html

```diff
@@ -6,17 +6,115 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <title>My Blog</title>
     <link
       href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
       rel="stylesheet"
       type="text/css"
     />
     <link rel="stylesheet" href="css/style.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <title>Virgo Fashion Magazine</title>
   </head>
-  <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+  <body>
+    <header>
+
+      <section class="top-container">
+        <p class="logo">VIRGO</p>
+        <section class="top-right">
+        <p class="top-right-p1">BRITAIN</p>
+        <p class="top-right-p2">SUBSCRIBE</p>
+        <img class="hburger-img" alt="Picture of a hamburger menu" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQhhwlrve66B8GHVhBRth6kDr6ml2SQR73R5g&usqp=CAU">
+        </section>
+      </section>
+
+      
+      <nav class="border-bottom">
+          <ul>
+            <li><a href="#">HOME</a></li>
+            <li><a href="#">FASHION</a></li>
+            <li><a href="#">RUNWAY</a></li>
+            <li><a href="#">BEAUTY</a></li>
+            <li><a href="#">NEWS</a></li>
+            <li><a href="#">SHOP</a></li>
+          </ul>
+      </nav>
+      
+    </header>
+
+    <main>
+     
+      <h3 class="stories" class="border-bottom">TOP STORY</h3>
+    
+      <div class="main-container">
+
+        
+        <section class="left-container">
+          <h3 class="h3">VIEWPOINT</h3>
+          <h1 class="h1">Black on Black trend continues</h1>
+          <h3 class="h3">BY: HEATHER MAY</h3>
+          <p class="date">23 October 2022</p>
+        </section>
+
+        <aside class="aside-container">
+          
+            <h3 class="stories">MAIN HEADLINES</h3>
+            <details>
+              <summary class="aside-border">Top outerwear this winter</summary>
+            </details>
+            <details>
+              <summary class="aside-border">Extra pointed toe the new pick</summary>
+            </details>
+            <details>
+              <summary class="aside-border">Accessorize with this type of head wear</summary>
+            </details>
+          
+        </aside>
+
+      </div>
+
+      <h3 class="stories">FASHION NEWS</h3>
+
+      <section class="article-container">
+        <article>
+          <h1>Styling tips for the puffer</h1>
+          <p>There are various ways to style the puffer jacket but there are different rules which depends on numerous
+            factors. What's the length, what's the fit and what about the color. <strong>Author: Mishy Mish</strong></p>
+          <button><a class="link"
```

Comment:
A link inside a button is not valid. A link should always have a href linking to part of a page or another url. A button will fire some other event such as submitting a form or opening a modal. Links are often designed to look like button but this shoul be done with css rather than a button element